functions-and-scope.js

```diff
@@ -72,6 +124,22 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+// Met een for loop alle waarden van de array langsgaan.
+// Check of het cijfer hoger is dan het vorige hoogste cijfer uit de array.
+// Als deze inderdaad hoger is wordt de variabele aangepast naar dit hoogste cijfer.
+
+function highestGrade(grades) {
+    let highestGrade = 0;
```

Comment:
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd 0 teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
    let highestGrade = grades[0];
```